examples/manifold/plot_tsne.py

```diff
@@ -0,0 +1,122 @@
+# Authors: Christopher Erick Moody <chrisemoody@gmail.com>
```

Comment:
The example is lacking a docstring.

It should be a somewhat complete chunk of text, with a title (the title is used in the gallery) that describes the purpose of the example, and what we can learn from it. And it should come first, before the author line.